web/client/components/contextcreator/ConfigurePluginsStep.jsx

```diff
@@ -205,7 +238,10 @@ const configurePluginsStep = ({
                         glyph: 'info-sign',
                         title: 'contextCreator.configurePlugins.searchResultsEmpty'
                     },
-                    onFilter: onFilterAvailablePlugins
+                    onFilter: onFilterAvailablePlugins,
+                    tools: [{
+                        id: "upload", label: <Glyphicon glyph="upload" />, onClick: () => onEnableUpload(true), tooltipId: 'contextCreator.configurePlugins.tooltips.upload'
```

Comment:
this tooltip seems not to be present, probably it is `uploadPlugin`, not upload. I can not see the icon too.
![image](https://user-images.githubusercontent.com/1279510/72910651-1ef4a080-3d39-11ea-81e4-1c9010d8a457.png)

Also, when I click I see this, that looks like some css is missing, isn't it? :
![image](https://user-images.githubusercontent.com/1279510/72910940-8c083600-3d39-11ea-9aba-a49c86946a96.png)

Reply:
* the icon and tooltip have been correct in the last commit
 * the current UI for the upload dropzone is not the final one, needs to be improved